kubernetes/services/nginx-storage.yml.j2

```diff
@@ -38,6 +38,8 @@ spec:
         - name: nginx-storage-config
           configMap:
             name: nginx-storage-config
+      securityContext:
+        fsGroup: 65534
```

Comment:
This needs some explanation (in comments). Why are you using `fsGroup` and why this particular value? Does this require the container to be built or configured in a specific way?